client/src/main.rs

```diff
@@ -65,7 +69,9 @@ fn main() -> Result<()> {
     env_logger::Builder::from_env(env_logger::Env::default().default_filter_or("info")).init();
 
     // Parse args
-    let args = Opt::from_args();
+    let mut args = Opt::from_args();
+    let anonymous_user = format!("anon{:04}", random_hack() % 10_000);
```

Comment:
What's this `random_hack` function do?

Reply:
Oh, just read that it creates a random hash. Weird function here though, might as well just call it `generate_hash` for professionality.

Reply:
It just generates a random number, and I didn't want to bother with the whole `rand` crate for that. It's kind of a hack since it's abuses the stdlib to get random numbers, but to be honest here it doesn't matter since it just makes a random name for an anonymous user